### ISSUE TITLE
archive nodejs/evangelism, remove @nodejs/evangelism team

### ISSUE THREAD

**Issue by Trott** [nodejs/admin#196]
I'm on the (unchartered) Evangelism WG and I'm pretty sure it doesn't actually exist in any meaningful way. It seems like it's role may be largely handled these days by CommComm and maybe some other groups. Time to archive the repo and remove the team?

/cc @nodejs/evangelism 

@nodejs/tsc @nodejs/community-committee

**Comment by codeekage** [comment]
@Trott we are still yet to come to a consensus on what we should do with the @ndoejs/evangelism WG.  

https://github.com/nodejs/community-committee/issues/275

**Comment by Trott** [comment]
@codeekage I could be wrong, but I don't think coming to consensus on that is the way to approach it. Either people are stepping up and doing the work or they're not. Why wait for CommComm approval? Why not start doing the work and asking for the things that are needed?

It's not clear to me that there should be or needs to be an Evangelism group that is separate from CommComm, but I won't second-guess CommComm if that's their determination.

I'll post this comment (slightly modified) over in that issue too.

**Comment by TimothyGu** [comment]
Yeah I agree with @Trott. Of course anyone "could" revive it (or merge it into another group), but the fact is that right now the group is unfortunately inactive.

We could always unarchive a repo in the future if the responsible individuals believe it to be useful.

**Comment by Trott** [comment]
@Gioyik @mikeal @rosskukulinski @vjk2005 With the removal of the evangelism team, you are currently on 0 @nodejs teams, so you may get an email notification that you've been removed from the org or something like that. You can ignore that (unless you are active in other areas and want to be added to those areas, in which case, leave a comment!). Thanks!
